File: 1212-sequential-digits/sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        ans=[]
        s="123456789"
        for i in range(0,10):
            for j in range(i+1,10):
                temp=s[i:j]
                if int(temp)>=low and int(temp)<=high:
                    ans.append(int(temp))
        return sorted(ans)            

File: 1212-sequential-digits/sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int>ans;
        string s ="123456789";
        int i,j;
        for(i=0;i<9;i++){
            for(j=i+1;j<9;j++){
                string temp = s.substr(i,j-i+1);
                int a = stoi(temp);
                if(a>=low&&a<=high){
                    ans.push_back(a);
                }
            }
        }
        sort(ans.begin(),ans.end());
        return ans;
    }
};